pkg/identity/cache/cell/metrics.go

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package identitycachecell
+
+import (
+	"time"
+
+	"github.com/cilium/cilium/pkg/metrics"
+	"github.com/cilium/cilium/pkg/metrics/metric"
+)
+
+type identityCacheMetrics struct {
+	TriggerLatency metric.Vec[metric.Observer]
+	TriggerFolds   metric.Vec[metric.Observer]
+	TimerDuration  metric.Vec[metric.Observer]
+}
+
+const subsystem = "identity_cache"
+
+func newIdentityCacheMetrics() *identityCacheMetrics {
```

Comment:
We should update the documentation to reflect that these metrics have been renamed (both in the upgrade notes and the metrics page)

Reply:
Done